subprojects/docs/src/docs/userguide/migration/upgrading_version_7.adoc

```diff
@@ -157,9 +157,9 @@ Providing this flag in the compiler options will disable this behaviour and allo
 For up-to-date checks Gradle relies on tracking the state of the inputs and the outputs of a task.
 Gradle used to ignore unreadable files in the input or outputs to support certain use-cases, although it cannot track their state.
 Declaring input or output directories on tasks which contain unreadable content has been deprecated and these use-cases are now supported by declaring the task to be untracked.
-Use the @link:{javadocPath}/org/gradle/api/tasks/UntrackedTask.html[UntrackedTask] annotation or the @link:{groovyDslPath}/org.gradle.api.Task.html#org.gradle.api.Task:doNotTrackState(java.lang.String)[Task.doNotTrackState()] method to declare a task as untracked.
+Use the @link:{javadocPath}/org/gradle/api/tasks/UntrackedTask.html[UntrackedTask] annotation or the link:{groovyDslPath}/org.gradle.api.Task.html#org.gradle.api.Task:doNotTrackState(java.lang.String)[Task.doNotTrackState()] method to declare a task as untracked.
```

Comment:
Should this `@link` be also removed? 

Also I see on some places there is `link{..}[...]` and somewhere is just link{..}[...] (without "`") is that ok?

Reply:
The problem here is that `@` renders as `@`. 
Before my changes:
![image](https://user-images.githubusercontent.com/423186/146770673-7821df2f-6a9d-4253-93ac-ad96015c6a74.png)

After my changes:
![image](https://user-images.githubusercontent.com/423186/146770761-1950d8d7-2067-4096-91bc-90de00798e48.png)

The `@` before the annotation is good, the `@` before the link to the method is bad.

Regarding \` or not \`:  I don't feel strongly if we put links to methods in \` or not. Though I agree that we probably should be consistent. I wouldn't change this as part of this commit, though.